### PR TITLE
Goliath Cleanup in preparation for merge to master

### PR DESCRIPTION
**Pull request by clee-r7**
This patch cleans up the Goliath branch in preparation for merge to master

### CODE REVIEW

File: lib/msf/core/session_manager.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ class SessionManager < Hash
 
   include Framework::Offspring
 
-  LAST_SEEN_INTERVAL = 60 * 2.5
+  LAST_SEEN_INTERVAL  = 60 * 2.5
```
Comment (mkienow-r7) [inline lib/msf/core/session_manager.rb]: Minor: extra space added

Comment (jbarnett-r7) [inline lib/msf/core/session_manager.rb]: Minor: extra space added

Comment (jbarnett-r7) [reply lib/msf/core/session_manager.rb]: LOL I literally did not see this comment when I made mine.

**Comment by jbarnett-r7** [comment]
LGTM. :shipit:

**Pull request merged**